apps/src/pythonlab/pyodideWorkerManager.ts

```diff
@@ -6,8 +6,13 @@ import {setLoadedCodeEnvironment} from '@cdo/apps/lab2/redux/systemRedux';
 import {MultiFileSource, ProjectFile} from '@cdo/apps/lab2/types';
 import {getStore} from '@cdo/apps/redux';
 
-import {parseErrorMessage} from './pythonHelpers/messageHelpers';
-import {MATPLOTLIB_IMG_TAG} from './pythonHelpers/patches';
+import {NeighborhoodSignal} from '../miniApps/neighborhood/types';
```

Comment:
nit: if the import is not in the same folder I like to use a fully-qualified import, ex. `@cdo/apps/miniApps/...`